sdk/src/client/api/block_builder/input_selection/mod.rs

```diff
@@ -408,13 +408,10 @@ impl InputSelection {
             return Err(Error::InvalidInputCount(self.selected_inputs.len()));
         }
 
-        let (remainder, storage_deposit_returns) = self.remainder_and_storage_deposit_return_outputs()?;
-
-        if let Some(remainder) = &remainder {
-            self.outputs.push(remainder.output.clone());
-        }
+        let (remainders, storage_deposit_returns) = self.remainder_and_storage_deposit_return_outputs()?;
 
         self.outputs.extend(storage_deposit_returns);
+        self.outputs.extend(remainders.iter().map(|r| r.output.clone()));
```

Comment:
Can you switch them just for consistency pls?

Reply:
I would rather have the remainder outputs last in the tx, because then it's easier to see what it does, do you want me to change the function then?

Reply:
Fair enough but yes pls change the function then 🙏🏻

Reply:
Changed

Reply:
I'm being annoying now but can we also rename the function to `storage_deposit_return_and_remainder_outputs`?